argus/action_manager/base.py

```diff
@@ -77,13 +77,17 @@ def check_cbinit_service(self, searched_paths=None):
         pass
 
     @abc.abstractmethod
-    def git_clone(self, repo_url, location):
-        """Clone from an remote repo to a specific location on the instance.
-
-        :param repo_url:
-            The remote repo url.
-        :param location:
-            Specific location on the instance.
+    def git_clone(self, repo_url, location, count, delay):
+        """Clone from a remote repository to a specified location.
+
+        :param repo_url: The remote repository url.
+        :param location: The target location for where to clone the repository.
```

Comment:
That would confuse the user, he would think that he needs to parse a folder name as a parameter, location is more precise in my opinion.